### PR TITLE
Visually separating add group by

### LINKED ISSUE #1177: Highlight the "Add group by" in the importer
image

Add a plus-sign and change background color of the cell.

### PR DESCRIPTION
**Pull request by abdelr**
Fixes #1177 (Visually separating add group by)

### CODE REVIEW

**Comment by abdelr** [comment]
Solved with a minimal visual changed. I would not like to change the code in the presenter to start differentiating when it is a add group by row and when it is not. Instead, we add the plus icon on the last column.

File: src/OSPSuite.UI/Views/Importer/ColumnMappingView.cs [hunk 1]
```diff
@@ -220,23 +220,13 @@ public override void InitializeBinding()
...
       {
          if (model.Source is AddGroupByFormatParameter)
-         {
-            if (string.IsNullOrEmpty(model.Source.ColumnName))
-            {
-               return _disabledRemoveButtonRepository;
-            }
-
             return _addButtonRepository;
-         }
```
Comment (Yuri05) [inline src/OSPSuite.UI/Views/Importer/ColumnMappingView.cs:239]: @abdelr Not sure what happens here. Can u explain?

Comment (abdelr) [reply src/OSPSuite.UI/Views/Importer/ColumnMappingView.cs:239]: The last column has the x icon for cleaning the row. For the special case of the add group by, it is meaningless so we added a plus instead (already configured in _addButtonRepository). So, just a visual change adding plus icon on add group by row, and keeping x icon for clear row

Comment (Yuri05) [reply src/OSPSuite.UI/Views/Importer/ColumnMappingView.cs:239]: ok, thx

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**